internal/wasm/sdk/opa/nop.go

```diff
@@ -0,0 +1,119 @@
+// Copyright 2021 The OPA Authors.  All rights reserved.
+// Use of this source code is governed by an Apache2
+// license that can be found in the LICENSE file.
+
+// +build !opa_wasm
+
+package opa
+
+import (
+	"context"
+
+	"github.com/open-policy-agent/opa/metrics"
+)
+
+// OPA is a stub implementation of a opa.OPA.
+type OPA struct {
+}
+
+// Result is a stub implementation of a opa.Result.
+type Result struct {
+	Result []byte
+}
+
+// EvalOpts is a stub implementation of a opa.EvalOpts.
+type EvalOpts struct {
+	Entrypoint int32
+	Input      *interface{}
+	Metrics    metrics.Metrics
+}
```

Comment:
I suppose we could pull Result and EvalOpts out of the build-flagged dichotomy...but it won't buy us that much.